editor/resources/meta.edn

```diff
@@ -19,6 +19,11 @@
    :help "Write log file to disk",
    :default false,
    :path ["project" "write_log"]}
+  {:type :string,
```

Comment:
Don't forget src/cm/dynamo/bob/meta.properties. Also add to docs/ project-settings.md

Reply:
I added it in build folder and didn't pay attention the file is missing in commit. thx